tests/foreman/ui/test_rhc.py

```diff
@@ -65,10 +66,32 @@ def fixture_setup_rhc_satellite(request, module_target_sat, module_rhc_org):
         module_target_sat.cli.Subscription.upload(
             {'file': manifests_path, 'organization-id': module_rhc_org.id}
         )
+        rh_repo1 = {
+            'name': constants.REPOS['rhel8_aps']['name'],
+            'product': constants.PRDS['rhel8'],
+            'reposet': constants.REPOSET['rhel8_aps'],
+            'basearch': constants.DEFAULT_ARCHITECTURE,
+            'releasever': '8',
+        }
+        rh_repo2 = {
+            'name': constants.REPOS['rhel7']['name'],
+            'product': constants.PRDS['rhel'],
+            'reposet': constants.REPOSET['rhel7'],
+            'basearch': constants.DEFAULT_ARCHITECTURE,
+            'releasever': '7Server',
+        }
```

Comment:
You can use directly `constants.REPOS['rhel7']` instead of `rh_repo2` (you are constructing the same dict structure which is already present in codebase)

Reply:
Updated the pr.

Reply:
or if you still prefer shorter name then assign it to your test vars:
```
rh_repo2 = constants.REPOS['rhel7']
```
oh another caveat: REPOS have arch while you use `basearch` - adapt to arch and then you are good

Reply:
oops, I didn't read it carefully.

Reply:
updated the pr.